python/lsst/obs/pfs/isrTask.py

```diff
@@ -110,13 +110,18 @@ class PfsIsrTaskConfig(ipIsr.IsrTaskConfig):
     brokenRedShutter = pexConfig.ConfigField(
         dtype=BrokenShutterConfig, doc="Broken shutter related configuration options."
     )
+    windowed = pexConfig.Field(dtype=bool, default=False,
+                               doc="Reduction of windowed data, for real-time acquisition? Implies "
+                               "overscanFitType=MEDIAN")
 
     def setDefaults(self):
         super().setDefaults()
         self.assembleCcd.retarget(PfsAssembleCcdTask)
 
     def validate(self):
         super().validate()
+        if self.windowed:
+            self.overscanFitType = "MEDIAN"
```

Comment:
Ditto.

Reply:
If we're putting this in the `IsrTask`, then why not have all `IsrTask` users set `isr.windowed` rather than `isr.overscanFitType`?